src/common-lib/urls/urls.ts

```diff
@@ -525,13 +542,25 @@ export const OAK_PAGES: {
     configType: "internal",
     pageType: "lesson-overview",
   }),
+  "lesson-overview-canonical": createOakPageConfig({
+    pathPattern: "/teachers/lessons/:lessonSlug",
+    analyticsPageName: "Lesson", // @todo this should be "Lesson (Canonical)"?
+    configType: "internal",
+    pageType: "lesson-overview-canonical",
+  }),
   "lesson-downloads": createOakPageConfig({
     pathPattern:
       "/teachers/programmes/:programmeSlug/units/:unitSlug/lessons/:lessonSlug/downloads",
     analyticsPageName: "Lesson Download",
     configType: "internal",
     pageType: "lesson-downloads",
   }),
+  "lesson-downloads-canonical": createOakPageConfig({
+    pathPattern: "/teachers/lessons/:lessonSlug/downloads",
+    analyticsPageName: "Lesson Download", // @todo this should be "Lesson Download (Canonical)"?
```

Comment:
@verityroberts @ewan-wakeman do we want to distinguish the canonical pages using the "Page name" prop? Or are we happy for it to be called "Lesson Download" regardless of whether the user is in a specific pathway?

Reply:
For now I will leave them as is. Let me know if you want to differentiate in the tracking plan